swim.go

```diff
@@ -182,7 +219,15 @@ func (s *SWIM) handle(msg pb.Message) {
 
 	switch msg.Payload.(type) {
 	case *pb.Message_Ping:
-		// handle ping
+		Address := s.config.BindAddress + ":" + string(s.config.BindPort)
```

Comment:
how about refactor to function this logic `handleXXX(msg pb.Message)` ?

Reply:
I agree